app/views/components/_banner.html.erb

```diff
@@ -0,0 +1,28 @@
+<%
+  title ||= false
+  aside ||= false
+%>
+
+<% if aside %>
+ <div class="app-c-banner app-c-banner--grid">
+  <div class="grid-row app-c-banner__aside">
+    <div class="column-third">
+      <p><%= aside %></p>
+    </div>
+
+    <div class="column-two-thirds app-c-banner__main">
+      <h2 class="app-c-banner__title"><%= title %></h2>
```

Comment:
@fofr At the moment, no, the only argument that is required is 'text'. I was trying to make the banner as general as possible, and couldn't see a reason why providing an aside and just text (with no title) would be a problem?